Failure to build with FF1705_L151CC using CMake

<!--

   ************************************** WARNING **************************************

   The ciarcom bot parses this header automatically. Any deviation from the 
   template may cause the bot to automatically correct this header or may result in a 
   warning message, requesting updates.

   Please ensure all sections of the template below are filled in and no changes 
   are made to the template format. Only bugs should be raised here as issues. 
   Questions or enhancements should instead be raised on our forums:
   https://forums.mbed.com/ .

   *************************************************************************************

-->

### Description of defect

<!--
    Add detailed description of what you are reporting.
    Good example: https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html
-->
```
Compiling example repos....

[EXAMPLES]> INFO     In folder 'mbed-os-example-blinky'
[EXAMPLES]> INFO     Compiling mbed-os-example-blinky FF1705_L151CC GCC_ARM
[EXAMPLES]> INFO     Executing command 'mbed-tools compile -t GCC_ARM -m FF1705_L151CC -c'...

#### STDOUT ####
Configuring project and generating build system...
-- The C compiler identification is GNU 9.2.1
-- The CXX compiler identification is GNU 9.2.1
-- The ASM compiler identification is GNU
-- Found assembler: /usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Python3: /usr/bin/python3.7 (found version "3.7.4") found components: Interpreter 
-- Checking for Python package prettytable -- found
-- Checking for Python package future -- found
-- Checking for Python package jinja2 -- found
-- Checking for Python package intelhex -- found
-- Configuring done
-- Generating done
-- Build files have been written to: /builds/workspace/mbed-os-ci_cmake-example-GCC_ARM-dev@7/examples/mbed-os-example-blinky/cmake_build/FF1705_L151CC/develop/GCC_ARM
Building Mbed project...
[1/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c
../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Config/RTX_Config.c:28:10: fatal error: cmsis_compiler.h: No such file or directory
   28 | #include "cmsis_compiler.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[2/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c
../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.c:26:10: fatal error: cmsis_compiler.h: No such file or directory
   26 | #include "cmsis_compiler.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[3/122] Building ASM object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/TOOLCHAIN_GCC/TARGET_M3/irq_cm3.S.obj
[4/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c.obj -c ../../../../mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c
../../../../mbed-os/cmsis/device/rtos/source/mbed_rtos_rtx.c:18:10: fatal error: cmsis.h: No such file or directory
   18 | #include "cmsis.h"
      |          ^~~~~~~~~
compilation terminated.
[5/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_tick_ptim.c:25:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[6/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c.obj -c ../../../../mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c
../../../../mbed-os/cmsis/device/rtos/source/mbed_rtx_handlers.c:18:10: fatal error: cmsis_compiler.h: No such file or directory
   18 | #include "cmsis_compiler.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[7/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c.obj -c ../../../../mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c
../../../../mbed-os/cmsis/device/rtos/TOOLCHAIN_GCC_ARM/mbed_boot_gcc_arm.c:18:10: fatal error: cmsis.h: No such file or directory
   18 | #include "cmsis.h"
      |          ^~~~~~~~~
compilation terminated.
[8/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/Source/os_systick.c:28:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[9/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/platform/source/DeepSleepLock.cpp.obj -c ../../../../mbed-os/platform/source/DeepSleepLock.cpp
In file included from ../../../../mbed-os/platform/source/DeepSleepLock.cpp:20:
../../../../mbed-os/platform/include/platform/mbed_atomic.h:22:10: fatal error: cmsis.h: No such file or directory
   22 | #include "cmsis.h"
      |          ^~~~~~~~~
compilation terminated.
[10/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/main.cpp.obj -c ../../../../main.cpp
In file included from ../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_storage.h:31,
                 from ../../../../mbed-os/rtos/./include/rtos/internal/mbed_rtos_storage.h:22,
                 from ../../../../mbed-os/rtos/./include/rtos/rtos.h:27,
                 from ../../../../mbed-os/mbed.h:23,
                 from ../../../../main.cpp:6:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[11/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_boot.c.obj -c ../../../../mbed-os/cmsis/device/rtos/source/mbed_boot.c
../../../../mbed-os/cmsis/device/rtos/source/mbed_boot.c:60:10: fatal error: cmsis.h: No such file or directory
   60 | #include "cmsis.h"
      |          ^~~~~~~~~
compilation terminated.
[12/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_thread.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[13/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mutex.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[14/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evflags.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[15/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_mempool.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[16/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c
../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_evr.c:27:10: fatal error: cmsis_compiler.h: No such file or directory
   27 | #include "cmsis_compiler.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[17/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_memory.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[18/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_semaphore.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[19/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_system.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[20/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_timer.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[21/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_delay.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[22/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_msgqueue.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[23/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c.obj -c ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c
In file included from ../../../../mbed-os/cmsis/device/RTE/include/RTE_Components.h:23,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_core_c.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_lib.h:30,
                 from ../../../../mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/rtx_kernel.c:26:
../../../../mbed-os/cmsis/device/rtos/include/mbed_rtx_conf.h:26:10: fatal error: mbed_rtx.h: No such file or directory
   26 | #include "mbed_rtx.h"
      |          ^~~~~~~~~~~~
compilation terminated.
[24/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusInOut.cpp.obj -c ../../../../mbed-os/drivers/source/BusInOut.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/DigitalInOut.h:20,
                 from ../../../../mbed-os/drivers/./include/drivers/BusInOut.h:20,
                 from ../../../../mbed-os/drivers/source/BusInOut.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[25/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalOut.cpp.obj -c ../../../../mbed-os/drivers/source/DigitalOut.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/DigitalOut.h:20,
                 from ../../../../mbed-os/drivers/source/DigitalOut.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[26/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BufferedSerial.cpp.obj -c ../../../../mbed-os/drivers/source/BufferedSerial.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/BufferedSerial.h:21,
                 from ../../../../mbed-os/drivers/source/BufferedSerial.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[27/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusIn.cpp.obj -c ../../../../mbed-os/drivers/source/BusIn.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/BusIn.h:20,
                 from ../../../../mbed-os/drivers/source/BusIn.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[28/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalIn.cpp.obj -c ../../../../mbed-os/drivers/source/DigitalIn.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/DigitalIn.h:20,
                 from ../../../../mbed-os/drivers/source/DigitalIn.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[29/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2C.cpp.obj -c ../../../../mbed-os/drivers/source/I2C.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/I2C.h:20,
                 from ../../../../mbed-os/drivers/source/I2C.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[30/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/CAN.cpp.obj -c ../../../../mbed-os/drivers/source/CAN.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/CAN.h:20,
                 from ../../../../mbed-os/drivers/source/CAN.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[31/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/DigitalInOut.cpp.obj -c ../../../../mbed-os/drivers/source/DigitalInOut.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/DigitalInOut.h:20,
                 from ../../../../mbed-os/drivers/source/DigitalInOut.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[32/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogIn.cpp.obj -c ../../../../mbed-os/drivers/source/AnalogIn.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/AnalogIn.h:20,
                 from ../../../../mbed-os/drivers/source/AnalogIn.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[33/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/BusOut.cpp.obj -c ../../../../mbed-os/drivers/source/BusOut.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/DigitalOut.h:20,
                 from ../../../../mbed-os/drivers/./include/drivers/BusOut.h:20,
                 from ../../../../mbed-os/drivers/source/BusOut.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[34/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp.obj -c ../../../../mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp
In file included from ../../../../mbed-os/platform/include/platform/mbed_power_mgmt.h:22,
                 from ../../../../mbed-os/cmsis/device/rtos/source/mbed_rtx_idle.cpp:26:
../../../../mbed-os/hal/include/hal/ticker_api.h:25:10: fatal error: device.h: No such file or directory
   25 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[35/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/I2CSlave.cpp.obj -c ../../../../mbed-os/drivers/source/I2CSlave.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/I2CSlave.h:20,
                 from ../../../../mbed-os/drivers/source/I2CSlave.cpp:17:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[36/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/AnalogOut.cpp.obj -c ../../../../mbed-os/drivers/source/AnalogOut.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/AnalogOut.h:20,
                 from ../../../../mbed-os/drivers/source/AnalogOut.cpp:18:
../../../../mbed-os/platform/include/platform/platform.h:28:10: fatal error: device.h: No such file or directory
   28 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[37/122] Building CXX object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj
FAILED: CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj 
/usr/local/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ @CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj.rsp -MD -MT CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj -MF CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj.d -o CMakeFiles/mbed-os-example-blinky.dir/mbed-os/drivers/source/FlashIAP.cpp.obj -c ../../../../mbed-os/drivers/source/FlashIAP.cpp
In file included from ../../../../mbed-os/drivers/./include/drivers/FlashIAP.h:29,
                 from ../../../../mbed-os/drivers/source/FlashIAP.cpp:28:
../../../../mbed-os/hal/include/hal/flash_api.h:23:10: fatal error: device.h: No such file or directory
   23 | #include "device.h"
      |          ^~~~~~~~~~
compilation terminated.
[38/122] Building C object CMakeFiles/mbed-os-example-blinky.dir/mbed-os/cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Library/cmsis_os1.c.obj
ninja: build stopped: subcommand failed.

#### STDERR ####
ERROR: CMake invocation failed!

More information may be available by using the command line option '-v'.

#### End of STDOUT/STDERR ####

[EXAMPLES]> INFO     In folder 'mbed-os-example-sockets'
[EXAMPLES]> INFO     In folder 'mbed-os-example-cellular'
[EXAMPLES]> INFO     In folder 'mbed-os-example-lorawan'
[EXAMPLES]> INFO     In folder 'mbed-os-example-devicekey'
[EXAMPLES]> INFO     In folder 'mbed-os-example-mbed-crypto/getting-started'
[EXAMPLES]> INFO     In folder 'mbed-os-example-nfc/NFC_EEPROM'
[EXAMPLES]> INFO     In folder 'mbed-os-example-nfc/NFC_SmartPoster'
[EXAMPLES]> INFO     In folder 'mbed-os-example-ble/BLE_Advertising'
[EXAMPLES]> INFO     In folder 'mbed-os-example-kvstore'


Passed Example Compilation:
+--------------+--------+-----------+----------+--------------+
| EXAMPLE NAME | TARGET | TOOLCHAIN | TEST GEN | BUILD RESULT |
+--------------+--------+-----------+----------+--------------+
+--------------+--------+-----------+----------+--------------+


Failed Example Compilation:
+------------------------+---------------+-----------+----------+--------------+
| EXAMPLE NAME           |     TARGET    | TOOLCHAIN | TEST GEN | BUILD RESULT |
+------------------------+---------------+-----------+----------+--------------+
| mbed-os-example-blinky | FF1705_L151CC |  GCC_ARM  |  UNSET   |    FAILED    |
+------------------------+---------------+-----------+----------+--------------+
Number of failures = 1
```

#### Target(s) affected by this defect ?

FF1705_L151CC

#### Toolchain(s) (name and version) displaying this defect ?

GCC_ARM (gcc-arm-none-eabi-9-2019-q4-major)

#### What version of Mbed-os are you using (tag or sha) ?
<!--
    For a released version please provide the release tag (this can be found as per the instructions below)

    mbed-os version can be found in /platform/mbed_version.h. The tag can be reconstructed as follows:
    mbed-os-MBED_MAJOR_VERSION.MBED_MINOR_VERSION.MBED_PATCH_VERSION
 
    Master branch is indicated by 'mbed-os-99.99.99
    
    For an issue found on Master please provide the sha being used.
-->
0e89f9d6d9ca - mbed-os (master)
28c085500eaf - mbed-os-example-blinky (development)

#### What version(s) of tools are you using. List all that apply (E.g. mbed-cli)

mbed-tools 7.3.0

#### How is this defect reproduced ? 


In folder 'mbed-os-example-blinky'
```
mbed-tools compile -t GCC_ARM -m FF1705_L151CC -c
```

--- a/targets/TARGET_STM/TARGET_STM32L1/TARGET_STM32L151xC/TARGET_XDOT_L151CC/CMakeLists.txt
+++ b/targets/TARGET_STM/TARGET_STM32L1/TARGET_STM32L151xC/TARGET_XDOT_L151CC/CMakeLists.txt
@@ -17,3 +17,6 @@ target_include_directories(mbed-xdot-l151cc
 )
 
 target_link_libraries(mbed-xdot-l151cc INTERFACE mbed-stm32l151xc)
+
+add_library(mbed-ff1705-l151cc INTERFACE)
+target_link_libraries(mbed-ff1705-l151cc INTERFACE mbed-xdot-l151cc)

